src/hooks/useDatabaseTankConfiguration.ts

```
@@ -30,7 +30,7 @@ export const useDatabaseTankConfiguration = (tanks: Tank[]): UseDatabaseTankConf
   // Load configuration from database on mount
   useEffect(() => {
     loadConfiguration();
-  }, []);
+  }, [loadConfiguration]);
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)